lib/cisco_node_utils/cmd_ref/fabricpath.yaml

```diff
@@ -0,0 +1,178 @@
+# fabricpath
+---
+# Fabricpath feature is not available on N3K and N9K
+_exclude: [/N3/, /N9K/]
+
+aggregate_multicast_routes:
+  _exclude:
+    - /N5K/
+  kind: boolean
+  config_get: "show run fabricpath"
+  config_get_token: '/^fabricpath multicast aggregate\-routes$/'
+  config_set: "<state> fabricpath multicast aggregate-routes"
+  default_value: false
+  
+allocate_delay:
+  kind: int
+  config_get: "show fabricpath timers"
```

Comment:
I'm hesitant to use feature commands instead of show run config. Is this the only way to get the state?